Packs/VirusTotal/Integrations/FeedLivehunt/FeedLivehunt.py

```diff
@@ -0,0 +1,266 @@
+import urllib3
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+class DetectionRatio:
+    malicious = 0
+    total = 0
+
+    def __init__(self, last_analysis_stats: dict):
+        self.malicious = last_analysis_stats['malicious']
+        self.total = last_analysis_stats['harmless'] + \
+            last_analysis_stats['suspicious'] + \
+            last_analysis_stats['undetected'] + \
+            last_analysis_stats['malicious']
+
+    def __repr__(self):
+        return f'{self.malicious}/{self.total}'
+
+
+class Client(BaseClient):
+    def build_iterator(self, limit: Optional[int] = 10,
+                       filter_: Optional[str] = None) -> List:
+        """Retrieves all entries from the feed.
+        Returns:
+            A list of objects, containing the indicators.
+        """
+
+        result = []
+
+        response = self.list_notifications_files(limit, filter_)
+
+        try:
+            for indicator in response.get('data', []):
+                result.append({
+                    'data': indicator,
+                    'type': 'file',
+                    'FeedURL': self._base_url
+                })
+        except ValueError as err:
+            demisto.debug(str(err))
+            raise ValueError(f'Could not parse returned data as indicator. \n\nError message: {err}')
+        return result
+
+    def list_notifications_files(
+            self,
+            limit: Optional[int] = None,
+            filter_: Optional[str] = None
+    ) -> dict:
+        """Retrieve VT Hunting Livehunt notifications files.
+        """
+        return self._http_request(
+            'GET',
+            'intelligence/hunting_notification_files',
+            params=assign_params(
+                filter=filter_,
+                limit=max(limit, 40),
```

Comment:
Seems like the feed is always pulling the same 20 indicators because of the limit argument in you http call. We want to get the updated indicators each fetch, it can be done by getting the new updated one from the API (if the API supports it), or we can get all indicators from the API and then Cortex XSOAR will filter the new indicators. You can take a look at FeedOffice365 for example. In each fetch they are getting all indicators, the limit is (-1) so it does not effect them.

Reply:
Yep, the API returns always the last indicators found in VT hunting, regardless whether they have been consumed before or not, unfortunately VT API doesn't support this. But, is this a problem? As the fetch ignore indicators that already exist in Cortex should't be a mayor problem, isn't it?